### DISCUSSION TITLE
buddhism/2024/mind_seal/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#78] [General]
# buddhism/2024/mind_seal/

https://shanenull.com/buddhism/2024/mind_seal/

**Comment by shane0** [comment]
Create a forum link here
Checklist brief and one with every chapter
Later three points on each chapter

**Reply by shane0** [reply]
- bodhi aspiring to enlightenment
- bodhi of subdue the mind
- bodhi of awakening the mind
- bodhi of progressive realization
- bodhi of perfect enlightenment

**Comment by shane0** [comment]
Riding the ox while looking for the ox

**Comment by shane0** [comment]
Dwell not in existence or emptiness
Pure non dual non conceptual awareness
Ever knowing in clarity in stillness
Inherent neither comes nor goes

**Comment by shane0** [comment]
## the meditative mind

|               | Meditative Mind | Untrained Mind |
| ------------- | --------------- | -------------- |
| Attention     | Focused         | Distracted     |
| Awareness     | Expanded        | Limited        |
| Thoughts      | Observing       | Identifying    |
| Emotions      | Regulating      | Reactive       |
| Perception    | Clear           | Clouded        |
| Judgment      | Non-judgmental  | Judgmental     |
| Response      | Intentional     | Automatic      |
| Concentration | Developed       | Scattered      |
| Insight       | Cultivated      | Absent         |
| Clarity       | Increased       | Decreased      |
| Equanimity    | Present         | Absent         |

**Comment by shane0** [comment]
Nurture the mind in stillness temperate it in action

**Comment by shane0** [comment]
chapter 6

false view

By contrast, the minds of sentient beings give rise to greed, anger, and ignorance every day, yearning for fortune, sensual pleasure, food, sleep, fame, and profit. Most people believe the purpose of living in this world is to pursue and enjoy these things.

**Comment by shane0** [comment]
"Neither increasing nor diminishing, the mind itself is the indestructible vajra. Regardless of the body's coming and going, the mind originally inheres in samadhi." Our deepest gratitude, Grand Master, for urging us to practice, not giving up on teaching us, and encouraging us to directly take on this very mind that is neither greater in the enlightened nor lesser in the mundane. Our deepest gratitude, Grand Master, for not forgoing strenuous toil in leading us by example to fearlessly tread the great bodhisattva path, and directly showing us the meaning of true samadhi: whether in action or stillness, during leisure or work, walking, standing, sitting, or resting, this very mind must be clear, lucid, unmoving in suchness, and integrating stillness and awareness as one.

From preface
